contract/index.js

```diff
@@ -0,0 +1 @@
+export * from './typeGuards.js';
```

Comment:
don't we have to add something like `"main": "./index.js"` to `package.json` for this to be useful?

Reply:
or maybe `index.js` is the default main thing?

Reply:
I'm getting...

```
(node:2612120) [DEP0151] DeprecationWarning: No "main" or "exports" field defined in the package.json for /home/connolly/projects/dapp-stake-control/node_modules/staking-contract/ resolving the main entry point "index.js", imported from /home/connolly/projects/dapp-stake-control/contract/test/stake.contract.test.ts
```